docs/Makefile

```diff
@@ -7,13 +47,17 @@ SPHINXBUILD   = sphinx-build
 SOURCEDIR     = .
 BUILDDIR      = _build
 
-# Put it first so that "make" without argument is like "make help".
-help:
-	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+# Build the "singlehtml" Sphinx target.  There are other supported targets
+# one could use; see `$(SPHINXBUILD) -M help` for a complete list.
+.PHONY: singlehtml
+singlehtml:
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS)
 
-.PHONY: help Makefile
+.PHONY: clean veryclean
```

Comment:
Do we need `veryclean` here when it's also phony-ed below where the target is declared?